src/reconstruct/plm_inline.hpp

```diff
@@ -47,12 +47,13 @@ void PiecewiseLinearX1(parthenon::team_mbr_t const &member, const int k, const i
 
   // compute L/R slopes for each variable
   for (int n = 0; n <= nu; ++n) {
-    parthenon::par_for_inner(member, il, iu, [&](const int i) {
-      // renamed dw* -> dq* from plm.cpp
-      dql(n, i) = (q(n, k, j, i) - q(n, k, j, i - 1));
-      dqr(n, i) = (q(n, k, j, i + 1) - q(n, k, j, i));
-      qc(n, i) = q(n, k, j, i);
-    });
+    parthenon::par_for_inner(DEFAULT_INNER_LOOP_PATTERN, member, il, iu,
```

Comment:
Why not pmb->par_for_inner? Maybe we should pass the meshblock in to the reconstruction routines?

Reply:
There were two reason
- from a practical point of view I didn't want to change the interface
- from a logical point of view there's no reason to use the `pmb->par_for_inner` as it's a 1:1 match to the `parthenon::par_for_inner(DEFAULT_INNER_LOOP_PATTERN,...`. This is different from the `pmb->par_for` where we want/need to use the `pmb` as the ExecutionSpace is tied to the MeshBlock. At the "inner" level there's no ExecutionSpace as the corresponding information is passed by the `member` variable.

Reply:
> At the "inner" level there's no ExecutionSpace as the corresponding information is passed by the member variable.

Ah, I don't think I appreciated this. In that case, perhaps `parthenon::par_for_inner(member, il, iu...` should still be supported? I.e., without the default loop pattern needing to be specified?

Reply:
Yes, that's a good point. I'll add this.